tests/test_applets/conservationTracking/testConservationTrackingHeadless.py

```diff
@@ -26,6 +26,7 @@
 import pytest
 import shutil
 import vigra
+import tempfile
```

Comment:
do you remember what change made the renames necessary? Commit message just says fix tests. Does this indicate a change of default behavior in file-naming?

Reply:
The generated nicknames for hierarchical data now also encompass the inner path, so that you can run ilastik on data.h5/dataset1 and data.h5/dataset2 without overwriting the results of the first with those of the latter.